### ISSUE TITLE
Add BedMachine Data to fetch

### ISSUE THREAD

**Issue by mdtanker** [mdtanker/antarctic_plots#26]
**Description of the desired feature:**

Add the Bedmachine dataset as a fetch call. Either figure out how to use authentication from NSIDC or use the following link from [here](https://github.com/ldeo-glaciology/pangeo-bedmachine/blob/master/load_plot_bedmachine.ipynb)

https://storage.googleapis.com/ldeo-glaciology/bedmachine/BedMachineAntarctica_2019-11-05_v01.nc

**Are you willing to help implement and maintain this feature?**

**Comment by mdtanker** [comment]
This [notebook](https://github.com/ldeo-glaciology/pangeo-glaciology-examples/blob/master/01_bedmachine.ipynb) uses Pangeo to load the data.

```python
from intake import open_catalog

cat = open_catalog('https://raw.githubusercontent.com/ldeo-glaciology/pangeo-bedmachine/master/bedmachine.yaml')
ds  = cat["bedmachine"].to_dask()
```

**Comment by mdtanker** [comment]
Closed for now with 3813d99c5c7a12449be3d91cc570b46f48c52a61, might eventually fetch directly from [NSIDC](https://nsidc.org/data/nsidc-0756/versions/1), but that will require figuring out how to ask users for login info during the `pooch.fetch` call.
